packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/DiagramRenderer.types.ts

```diff
@@ -17,10 +17,10 @@ import { CustomEdgeData } from './edge/CustomEdge.types';
 
 export interface DiagramRendererProps {
   diagram: Diagram;
+  autoLayoutElk: boolean;
```

Comment:
just `autoLayout`

Reply:
There already a variable with this same  const { autoLayout } = useLayout();

Reply:
`isAutoLayout` then :)